server/apps/pa_img/pa_img_datalayer.py

```diff
@@ -119,15 +135,13 @@ def find(self, resource, req, lookup):
                         fields['photos'] = 'true'
 
         if not fields:
```

Comment:
Please change the code here (https://dev.sourcefabric.org/browse/SDPA-52):
if not fields:
+            url += '/latest'
+            fields['ck'] = 'public'
+            fields['days_since'] = 5
+        else:
+            url += '/search'
+            fields['ck'] = 'superdesk'